src/librbd/ImageCtx.cc

```diff
@@ -59,7 +59,7 @@ class ThreadPoolSingleton : public ThreadPool {
                  "rbd_op_threads") {
     start();
   }
-  virtual ~ThreadPoolSingleton() {
+  ~ThreadPoolSingleton() {
```

Comment:
Thanks @dillaman @cbodley , I will update this PR to `~destructor() override` soon.

@liewegas need we unify other PR?